huxley/core/migrations/0002_auto_20150802_1835.py

```diff
@@ -33,6 +33,12 @@ class Migration(migrations.Migration):
         ),
         migrations.AddField(
             model_name='school',
+            name='assignments_finalized',
```

Comment:
Yeah So I messed up my migrations on a pull and to fix it I deleted the previous 0002 and created a new squashed migration. Upon rethinking it this will cause problems for anyone who had the previous 0002 migration. Fixing now.